concepts/Common/Licensing/25 How to Use the License Key.md

```diff
@@ -0,0 +1,7 @@
+The license key must be specified in `GlobalConfig`. You can use the same license key for all your applications.
+
+    <!-- tab: config.js -->
```

Comment:
What does `config.js` mean? I'm not sure how this tab (?) will be displayed on the site, but it's important to avoid confusion - config is a function, not a file